cl/search/tests/tests_es_recap.py

```diff
@@ -1068,9 +1081,12 @@ def test_party_name_and_children_filter(self) -> None:
 
         ## The party_name and attorney filter can constrain the results
         # returned, along with string query, parent and child filters.
+        # Note that the case 'California v. America' is not found here.
+        # Dockets without filings cannot be located by parties fields if
+        # combined with any other filter or string query.
```

Comment:
Would this work better as an assertion with a comment?

Reply:
Sure I've added the assertion and the comment.
- Also added an additional test to confirm that it is possible to match a docket without filings, as in this case, by using a fielded search and a docket-level field filter.
e.g:
```
"q": attorney:(Harris Martin)",
"case_name": "California",
```